frontend/app/forms/bank/form-bank.component.js

```diff
@@ -170,14 +170,22 @@ export default class FormBank extends React.Component {
             />
           )}
         />
+        <Route
+          path={`${this.props.match.url}/proof-of-completed-service`}
+          render={props => (
+            <GovernmentForm
+              name="Proof of Completed Service"
+              WebForm={ProofOfCompletedService_Web}
+              PdfForm={ProofOfCompletedService_Pdf}
+        />
         <Route
           path={`${this.props.match.url}/petition-for-expungement-from-bci`}
           render={props => (
             <GovernmentForm
               name="Petition for Expungement (Special Certificate from BCI)"
               WebForm={PetitionForExpungementFromBci_Web}
               PdfForm={PetitionForExpungementFromBci_Pdf}
-            />
+         />
```

Comment:
I'm pretty sure that when you merge master in to fix the build, that you'll get a new build failure for the formatting of this code. We use a tool called [Prettier](https://prettier.io/) that auto-formats our code so we don't have to worry about doing it ourselves. We also use [husky](https://github.com/typicode/husky) and [pretty-quick](https://github.com/azz/pretty-quick) which set up prettier automatically for all people contributing to the codebase. The following code in our package.json shows this:

https://github.com/JustUtahCoders/utahexpungements.org/blob/6386186e1f0650eed09249e89313551f38e7bccd/package.json#L17-L21

What I'm trying to show here is that you seem to have accidentally bypassed all of this when you set up your local version of the code. Try the following to fix:

```sh
rm -rf node_modules # this doesn't work on windows, but just delete the entire node_modules directory
yarn install
yarn prettier 'frontend/**/*' --write

# now add and commit
git commit -a -m "Fixing code formatting"
```

^ Note that you normally don't have to do these steps, but that your local version of the code somehow bypassed all of this. The normal way to bypass is to do `git commit --no-verify` intentionally, which I doubt you did. So it must have been a problem with the installation that prevented the git hook from working.